frontend/src/js/constants/Constants.js

```diff
@@ -107,15 +108,149 @@ const JOYRIDE_STEPS = [
             "Use this as an overlay or as an in-cell side-by-side comparison.",
             '</br><span style="color: red;">Clear Cell</span> will reset the cell back to the default. ',
             "This can be undone if you accidentally click it with the undo button.",
-            '</br><span style="color: blue;">Colormap Editor</span> will open a window for creating, editing, and applying colormaps.'
+            '</br><span style="color: blue;">Colormap Editor</span> will open a window for creating, editing, and applying colormaps.',
+            '</br><span style="color: purple;">Export</span> allows you to export/save the plot. NOT Implemented.',
+            '</br><span style="color: orange;">Calculator</span> allows you to create a new variable based on',
+            ' calculations performed on another variable.'
         ),
         selector: ".tools-container",
         position: "right",
         type: "click"
+    },
+    {
+        title: "Calculator",
+        text: "This button opens the calculator tool! Cool right?! Yeah it's cool :p",
+        selector: "#open-calculator-button",
+        position: "right",
+        type: "hover"
     }
 ];
 
+const CALCULATOR_STEPS = [
+    {
+        title: 'Calculator Tour',
+        text: 'This is a tour of the calculator and how to use it!',
+        selector: '#calculator-main',
+        position: 'left',
+        type: 'click'
+    },
+    {
+        title: 'Variables List',
+        text: "All the available variables that you've loaded are listed here.",
+        selector: '#calc-variable-region',
+        position: 'right',
+        type: 'click'
+    },
+    {
+        title: 'Derived Variable',
+        text: "Enter the name of the new variable you wish to derive here.",
+        selector: '#new-variable-name',
+        position: 'left',
+        type: 'click'
+    },
+    {
+        title: 'Calculation Area',
+        text: 'The calculation to be performed will be expressed here.'.concat(
+            '<br />To enter a variable, drag it outside of this area and within the green rectangle that appears.'),
+        selector: '#calculation',
+        position: 'right',
+        type: 'click'
+    },
+    {
+        title: 'Calculator Operations',
+        text: "The calculator supports TWO operands (constants or variables) and ONE operator, when deriving a new variable.".concat(
+            "However you can use previously derived variables to perform further calculations."),
+        selector: '#calc-button-region',
+        position: 'bottom-right',
+        type: 'hover'
+    },
+    {
+        title: 'Regrid Operations',
```

Comment:
Can we show the drop down?

Reply:
I think its possible, but will take some working out how exactly to do it. It might be possible to add a method to the ```step:before``` that would set the dropdown as open.

https://github.com/gilbarbara/react-joyride/issues/133

Reply:
I'll see what I can figure out for the drop-down.

Reply:
Showing the drop down would be complicated to implement and it is not very clear on how to do it from what I've seen. I can keep working at it to see if I find a way, but maybe I should leave this for later?